examples/website/geospatial/app.tsx

```diff
@@ -175,14 +181,14 @@ export default function App(props: AppProps) {
         onViewStateChange={({viewState}) => setState((state) => ({...state, viewState}))}
         controller={{type: MapController, maxPitch: 85}}
         getTooltip={({object}) => {
-          const {name, ...properties} = object?.properties || {};
+          const {name, ADMIN, ...properties} = object?.properties || {};
```

Comment:
Maybe best to define the possibility of overriding the tooltop function per dataset, rather than hacking in dataset specific prop names into the common tooltip function?

Reply:
Done